tensorflow/python/ops/array_ops.py

```diff
@@ -308,6 +308,45 @@ def listdiff(x, y, out_idx=None, name=None):
                         "Please switch to tf.sets.difference().")
 @tf_export(v1=["setdiff1d"])
 def setdiff1d(x, y, index_dtype=dtypes.int32, name=None):
+  """Computes the difference between two lists of numbers or strings.
+
+  Given a list x and a list y, this operation returns a list out that
+  represents all values that are in x but not in y. The returned list
+  out is sorted in the same order that the numbers appear in x
+  (duplicates are preserved). This operation also returns a list idx
+  that represents the position of each out element in x.
+
+  In other words:
+
+  ```python
+  out[i] = x[idx[i]] for i in [0, 1, ..., len(out) - 1]
+  ```
+
+  For example, given this input:
+  ```python
+  x = [1, 2, 3, 4, 5, 6]
+  y = [1, 3, 5]
+  ```
+
+  This operation would return:
+
+  ```python
```

Comment:
As Alex said, it would be nice to have this in the [doctest](https://docs.python.org/3/library/doctest.html) format.

Please remove the ```python and ``` after you add `>>>`.

Try the code out locally on your machine with TF2 rc2 and python3.